frontend/packages/metal3-plugin/src/components/host-detail.tsx

```diff
@@ -161,6 +163,18 @@ const BaremetalHostDetails: React.FC<BaremetalHostDetailsProps> = ({
                 <dd>{_.filter([manufacturer, productName]).join(', ')}</dd>
               </>
             )}
+            {bios && (
```

Comment:
The default return value from the selector is truthy so it'll be true here but then on line 170, it'll fail to find the `version` property. I wonder if the selector should just omit the default value?

Reply:
Thanks for the pointers, removed the default value that would otherwise be returned with _get(), which will now return 'undefined'.